core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -323,7 +323,9 @@ message FileInfo {
 }
 
 message GetFilePathPResponse {
+  /* For backward compatibility, remove path in the next release version. */
   optional string path = 1;
+  optional FileInfo fileInfo = 2;
```

Comment:
We already have `FileSystemMasterClient#getStatus` for the same thing. Should not duplicate RPCs that do the same thing.